eps/README.md

```diff
@@ -0,0 +1,106 @@
+# Enhancement Proposal (EP)
+
+This document describes the process of submitting an Enhancement Proposal (EP). EPs are a way to propose, communicate,
+and coordinate on new features for the NGINX Kubernetes Gateway. It is based off of
+the [Gateway Enhancement Proposals](https://github.com/kubernetes-sigs/gateway-api/blob/main/geps/overview.md). Their
+purpose is to:
+
+- Make changes and proposals discoverable (current and future).
+- Provide a common place to discuss design, architecture, and impacts of a particular change.
+- Document design ideas, tradeoffs, and decisions for historical reference.
+
+## Process
+
+The diagram below shows the EP process:
+
+```mermaid
+flowchart TD
+    D([Open Discussion]) --> C
+    C([Issue Created]) --> Provisional
+    Provisional -->|EP Doc PR<br />done| Implementable
+    Provisional -->|If practical <br /> work needed| Prototyping
+    Prototyping -->|EP Doc PR<br />done| Implementable
+    Implementable -->|Work completed| Completed
+```
+
+### 1. Open a GitHub Discussion
+
+Before creating an issue or EP, open
+a [GitHub discussion](https://github.com/nginxinc/nginx-kubernetes-gateway/discussions). Describe the feature you would
+like, any use cases you have, and other relevant details. Beginning with a discussion allows you to get feedback from
+the maintainers and the community before you invest time in writing an EP.
+
+### 2. Create an Issue
+
+If there is consensus on the discussion post that the feature is important and should be included in the roadmap, a
+maintainer will ask you
+to [open an issue](https://github.com/nginxinc/nginx-kubernetes-gateway/issues/new?assignees=&labels=proposal&projects=&template=enhancement.md&title=)
+on GitHub.
+
```

Comment:
If we come to an agreement in the issue, could the process stop here?  And the next step is a PR with the enhancement?
Is this the responsibility of maintainer(s) to decide if this should be captured in an EP prior to implementation?

Reply:
I added a note to this section about how not all issues require EPs.